rest/route/github_test.go

```diff
@@ -302,11 +302,18 @@ func (s *GithubWebhookRouteSuite) TestRetryCommentTrigger() {
 	commentString := issueComment.Comment.GetBody()
 	s.Equal(retryComment, commentString)
 
-	s.True(triggersPatch("created", commentString))
-	s.False(triggersPatch("deleted", commentString))
+	s.True(triggersPatch(commentString))
 
 	//test whitespace trimming
-	s.True(triggersPatch("created", "  evergreen retry "))
+	s.True(triggersPatch("  evergreen retry "))
+}
+
+func (s *GithubWebhookRouteSuite) TestRefreshStatusTrigger() {
+	s.True(triggersPatch(refreshStatusComment))
```

Comment:
I think these assertions may need to be flipped in this test?

Reply:
was calling the wrong function (#copypasta) but fwiw no need for reviewers to help with debugging failing tests, since that should be on the writer